docs/guides/cloud.rst

```diff
@@ -0,0 +1,133 @@
+.. _ref_guide_cloud:
+
+============
+EdgeDB Cloud
+============
+
+:edb-alt-title: Using EdgeDB Cloud (Beta)
+
+.. warning::
+
+    EdgeDB Cloud is currently in beta. This guide will not work unless you have
+    access. To request access, click the "Join Cloud Waitlist" button below or
+    join `our Discord <https://discord.gg/umUueND6ag>`_ and request access
+    there.
+
+.. raw:: html
+
+    <a class="pageNav_promoLink__OSucb" href="/p/cloud-waitlist"
+      style="width:152px;margin-left:0;text-align:center;">
+      <span>Join Cloud Waitlist</span>
+    </a>
+
+EdgeDB Cloud is the easiest way to host your EdgeDB instance. You have two
+options for interfacing with EdgeDB Cloud: via our CLI or through a graphical
+web interface.
+
+CLI Cloud Interface
+===================
+
+To use Cloud via the CLI, first log in using :ref:`ref_cli_edgedb_cloud_login`.
+Once your login is successful, you will be able to create an instance using
+either :ref:`ref_cli_edgedb_instance_create` or
+:ref:`ref_cli_edgedb_project_init`, depending on whether you also want to
+create a local project linked to your instance.
+
+In either case, you will pass an instance name matching the EdgeDB Cloud
+instance name format: ``<github-username>/<instance-name>``
```

Comment:
we should document the instance name requirements somewhere, see https://github.com/edgedb/shared-client-testcases/pull/27 for the canonical list

Reply:
* instance names cannot start with a dash `-` or contain double dashes `--`
* only latin alpha-numeric characters, `_`, or `-`
* maximum instance name length is 61 characters minus the length of your GitHub username

Reply:
Devon, we're going to be rapidly iterating the cloud UI. Let's drop the screenshots entirely.

Reply:
@1st1 Based on my read of @fantix's notes and the test cases in the [PR](https://github.com/edgedb/shared-client-testcases/pull/27) @zackelan referenced, it looks like underscore is also now disallowed. Can someone confirm or deny?